### PR TITLE
chore(#8692) flaky navigation hamburguer menu test

### PR DESCRIPTION
**Pull request by lorerod**
# Description

medic/cht-core #8692 
# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs

If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8692-flaky-navigation-hamburguer-menu-test/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8692-flaky-navigation-hamburguer-menu-test/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8692-flaky-navigation-hamburguer-menu-test/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Review by tatilepizs — APPROVED** [review]
Thank you @lorerod for cleaning the code, it looks really good! ✨

**Comment by lorerod** [comment]
Hi reviewers! upgrade e2e is failing. I am trying to reproduce the error and, I will you know when it is really ready for a re-review.
Thanks.

File: tests/e2e/upgrade/wdio.conf.js [hunk 1]
```diff
@@ -118,11 +118,11 @@ const servicesStartTimeout = () => {
...
 const upgradeConfig = Object.assign(wdioBaseConfig.config, {
-  specs: 
-   [
-     'upgrade.wdio-spec.js',
-     '*.wdio-spec.js'
-   ],
+  specs:
+    [
+      'upgrade.wdio-spec.js',
+      //'*.wdio-spec.js'
```
Comment (garethbowen) [inline tests/e2e/upgrade/wdio.conf.js:124]: Was this on purpose?

**Comment by garethbowen** [comment]
@lorerod I've noticed the upgrade test failing for me too but haven't had a chance to look into it. I'll try and get to it soon...

File: tests/e2e/upgrade/wdio.conf.js (hunk 1 above)
Comment (lorerod) [reply tests/e2e/upgrade/wdio.conf.js:124]: Just trying to get a screenshot of the actual fail step.

Comment (lorerod) [reply tests/e2e/upgrade/wdio.conf.js:124]: I will revert this.

**Comment by lorerod** [comment]
@garethbowen I think I'm on to something here with the upgrade e2e. This is the screenshot when failing:
81f7bf7ce4f2a6c5
It doesn't look like the new About page. I added a reload step. Waiting for checks to finish.

File: tests/e2e/upgrade/upgrade.wdio-spec.js [hunk 2]
```diff
@@ -46,18 +46,6 @@ const deleteUpgradeLogs = async () => {
...
-  await commonPage.openHamburgerMenu();
-  await (await commonPage.logoutButton()).waitForClickable();
-  await (await commonPage.logoutButton()).click();
-  $('.modal-dialog .modal-body').waitForDisplayed();
-  const submitBtn = $('.modal-dialog a.btn.submit');
-  await (await submitBtn).waitForClickable();
-  await (await submitBtn).click();
-  await browser.pause(100); // Wait for login page js to execute
-};
-
```
Comment (garethbowen) [inline tests/e2e/upgrade/upgrade.wdio-spec.js:60]: The function below this has a comment that it should be removed too though I'm not sure what to replace it with.

File: webapp/src/ts/modules/about/about.component.html [hunk 3]
```diff
@@ -113,15 +117,19 @@
...
           </div>
 
           <div>
             <label translate>replication.last.success.from</label>
-            <p *ngIf="!replicationStatus.disabled && replicationStatus.lastSuccessFrom">{{replicationStatus.lastSuccessFrom | simpleDateTime}}</p>
-            <p *ngIf="!replicationStatus.disabled && !replicationStatus.lastSuccessFrom" translate>replication.last.success.unknown</p>
+            <p *ngIf="!replicationStatus.disabled && replicationStatus.lastSuccessFrom">
+              {{replicationStatus.lastSuccessFrom | simpleDateTime}}</p>
+            <p *ngIf="!replicationStatus.disabled && !replicationStatus.lastSuccessFrom" translate>
+              replication.last.success.unknown</p>
```
Comment (garethbowen) [inline webapp/src/ts/modules/about/about.component.html:132]: There are a lot of line breaks being added which makes review difficult. Are these being added automatically by your editor?

File: tests/page-objects/default/about/about.wdio.page.js [hunk 4]
```diff
@@ -15,9 +16,17 @@ const getVersion = async () => {
   return await (await version()).getText();
 };
 
+const reload = async () => {
+  await (await $(RELOAD_BUTTON)).waitForClickable();
+  await (await $(RELOAD_BUTTON)).click();
+  await (await $(RELOAD_BUTTON)).waitForDisplayed();
+};
+
```
Comment (garethbowen) [inline tests/page-objects/default/about/about.wdio.page.js:24]: I don't think this is called from anywhere. Can we revert this?

File: tests/utils/index.js [hunk 5]
```diff
@@ -1079,7 +1079,7 @@ const waitForDockerLogs = (container, ...regex) => {
 
   const promise = new Promise((resolve, reject) => {
     timeout = setTimeout(() => {
-      console.log('Found logs', logs, 'watched for', ...regex);
+      console.log('Found logs', logs, 'did not match expected regex:', ...regex);
```
Comment (garethbowen) [inline tests/utils/index.js:1082]: This is only printed when the timeout expires, but the log message was confusing to me.

**Review by garethbowen — APPROVED** [review]

**Review by latin-panda — APPROVED** [review]
LGTM!

**Pull request merged**